### PR TITLE
Setup now creates semantifly binary

### PR DESCRIPTION
**Pull request by astradzhao**

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -1,7 +1,21 @@
...
 go install google.golang.org/protobuf/cmd/protoc-gen-go@latest
-mkdir -p ~/opt && mkdir -p ~/opt/semantifly && cd ~/opt/semantifly
+mkdir -p ~/opt/semantifly && cd ~/opt/semantifly
+
+# Download and install protoc
 wget https://github.com/protocolbuffers/protobuf/releases/download/v27.3/protoc-27.3-linux-x86_64.zip
 unzip protoc-27.3-linux-x86_64.zip
-export PATH=$PATH:~/opt/semantifly/bin
-echo "Protoc installed to ~/opt/semantifly/bin: fix pending to add to PATH"
-echo "For now, invoke as eg '~/opt/semantifly/bin/protoc -I=src/proto --go_out=src/proto src/proto/index.proto'"
```
Comment (fredxfred) [inline setup.sh:7]: We still want to run this protoc command even though technically we don't yet link the generated go files. 

Ideally, we would run this command for all .proto files in src/proto and not just index.proto

File: setup.sh [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+
+# Build binary, move to PATH
+go build -o ~/opt/semantifly/semantifly .
+echo 'export PATH="$HOME/opt/semantifly:$PATH"' >> ~/.bashrc
+
+# Update PATH
+. ~/.bashrc
+
+cd -
+find src/proto -name "*.proto" -exec ~/opt/semantifly/bin/protoc -I=src/proto --go_out=src/proto {} \;
```
Comment (fredxfred) [inline setup.sh]: This needs to be performed before the go build command since we want to link against the generated pb.go files to build semantifly

Comment (astradzhao) [reply setup.sh]: fixed

File: setup.sh (hunk 1 above)
Comment (astradzhao) [reply setup.sh:7]: fixed

**Pull request merged**